a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -297,50 +298,38 @@ def execute(self, vthunder, loadbalancer, added_ports, subnet, ifnum_master=None
                                                                          ifnum_address[ifnum],
-                                                                         subnet.ip_version)
-                        elif interfaces['interface']['ethernet-list'][i]['action'] == "disable":
-                            self.axapi_client.system.action.setInterface(ifnum, None,
-                                                                         subnet.ip_version)
+                                                                         6)
+                        else:
+                            self.axapi_client.system.action.setInterface(ifnum, None, 4)
                     LOG.debug("Configured the ethernet interface for vThunder: %s", vthunder.id)
                 else:
                     for i in range(len(interfaces['interface']['ethernet-list'])):
                         ifnum = interfaces['interface']['ethernet-list'][i]['ifnum']
-                        if subnet.ip_version == 6:
-                            if not ifnum_address[ifnum]:
-                                LOG.warning('IPv6 Address for ethernet %s not found'
-                                            'in configurations', ifnum)
-                                continue
+                        if ifnum_address.get(ifnum):
                             if backup_vthunder:
                                 self.axapi_client.device_context.switch(2, None)
                                 self.axapi_client.system.action.setInterface(ifnum,
                                                                              ifnum_address[ifnum],
-                                                                             subnet.ip_version)
+                                                                             6)
                             else:
                                 self.axapi_client.device_context.switch(1, None)
                                 self.axapi_client.system.action.setInterface(ifnum,
                                                                              ifnum_address[ifnum],
-                                                                             subnet.ip_version)
-                        elif interfaces['interface']['ethernet-list'][i]['action'] == "disable":
-                            self.axapi_client.device_context.switch(1, None)
-                            self.axapi_client.system.action.setInterface(ifnum, None,
-                                                                         subnet.ip_version)
-                            self.axapi_client.device_context.switch(2, None)
-                            self.axapi_client.system.action.setInterface(ifnum, None,
-                                                                         subnet.ip_version)
+                                                                             6)
+                    else:
```

Comment:
please check the indent for the else block. I assume it should use the same indent as line 317

Reply:
Happened by mistake while resolving flake8 issues, Corrected it.
Thanks